src/mapport.cpp

```diff
@@ -0,0 +1,137 @@
+// Copyright (c) 2020 The Bitcoin Core developers
```

Comment:
In commit "refactor: Move port mapping code to own files"

Nit: when you're copying/moving code, retain the copyright years of the original code.

Reply:
The UPnP support code has been added in 2011 (see #133).
Is it correct to retain pre-2011 copyright years?

Could it be "2011-2020" in new files?

Reply:
See the latest [push](https://github.com/bitcoin/bitcoin/pull/18077#issuecomment-591285675).